### PR TITLE
About Dialog - Add link to documentation

### PR DESCRIPTION
**Pull request by yodem**
Fixes: #134
Documentation added to About dialog:

Documentation2

### CODE REVIEW

**Comment by sgratch** [comment]
@bond95 @sjd78 please review
BTW, VM portal documentation is not up to date, so I guess we need to open an issue for updating it with 4.3 guide https://www.ovirt.org/documentation/intro-vm-portal/intro-vm-portal.html

**Comment by yodem** [comment]
@sjd78 can you please review?

File: src/intl/messages.js [hunk 1]
```diff
@@ -149,6 +149,7 @@ export const messages: { [messageId: string]: MessageType } = {
   disksCardEditTooltip: 'Edit Disks for {vmName}',
   diskSizeHasToBeAPositiveInteger: 'Disk size has to be a positive integer.',
   displayAll: 'Display all',
+  documentation: 'For further information see {link}',
```
Comment (sjd78) [inline src/intl/messages.js]: s/documentation/aboutDialogDocumentationLink/

Comment (sjd78) [reply src/intl/messages.js]: Add key `aboutDialogDocumentationText: 'Documentation',`

File: src/components/About.js [hunk 2]
```diff
@@ -43,6 +43,8 @@ class AboutDialog extends React.Component {
       apiVersion = `${oVirtApiVersion.get('major')}.${oVirtApiVersion.get('minor')}`
     }
 
+    const message = (<span dangerouslySetInnerHTML={{ __html: `${msg.documentation({ link: `<strong><a href=${fixedStrings.DOCUMENTATION_LINK} target='_blank' id=${idPrefix}-documentation-link>Documentation</a></strong>` })}` }} />)
```
Comment (sjd78) [inline src/components/About.js]: Need quotes around a few more attributes on the `a` tag...
```jsx
    const docLink = msg.aboutDialogDocumentationLink({ 
      link: `<strong><a href='${fixedStrings.DOCUMENTATION_LINK}' target='_blank' id='${idPrefix}-documentation-link'>${msg.aboutDialogDocumentationText()}</a></strong>` 
    })
```

and later, put the `span` more inline with the other markup
```jsx
    { fixedStrings.DOCUMENTATION_LINK &&
      <li id={`${idPrefix}-documentation`}>
        <span dangerouslySetInnerHTML={{ __html: docLink }} />
      </li>
    }
```

also, need to pull "Documentation" out of the static text and add a new key for it...

**Comment by isaranova** [comment]
ci build please

**Comment by isaranova** [comment]
LGTM

**Pull request merged**